custom_components/tahoma/__init__.py

```diff
@@ -106,17 +105,15 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
     # Map Overkiz device to Home Assistant platform
     for device in coordinator.data.values():
+        _LOGGER.debug(
+            "The following device has been retrieved. Report an issue if not supported correctly (%s)",
+            device,
+        )
         platform = OVERKIZ_DEVICE_TO_PLATFORM.get(
             device.widget
         ) or OVERKIZ_DEVICE_TO_PLATFORM.get(device.ui_class)
         if platform:
             platforms[platform].append(device)
-            log_device("Added device", device)
-        elif (
-            device.widget not in IGNORED_OVERKIZ_DEVICES
-            and device.ui_class not in IGNORED_OVERKIZ_DEVICES
```

Comment:
There is the same logic in platforms too.

Either :
- keep as it is
- remove all use of `IGNORED_OVERKIZ_DEVICES`